### PR TITLE
Implement password check

### PR DESCRIPTION
**Pull request by Wamacava**
Hi,

Just to let you know, the only changes that I did are passing the password to the Customer's constructor and comparing it to the one passed to the checkLogInDetails function.

### CODE REVIEW

File: NewBank/newbank/server/Customer.java [hunk 1]
```diff
@@ -3,39 +3,50 @@
...
-			fromAccount.removeMoney(amount);
-			toAccount.addMoney(amount);
-			return true;
-		}
-	return false;
-	}
+
+    private ArrayList<Account> accounts;
+
+    private String password;
```
Comment (Wamacava) [inline NewBank/newbank/server/Customer.java:9]: Sorry for adding a lot of white spaces changes. IntelliJ doesn't like tabs, I don't know how to force it not to change tabs to spaces. 

I'll mark all significant changes in this file. This is one of them.

**Review by acairns-345 — APPROVED** [review]
Hi Lukasz, thanks for your work on this! All approved - I've run it and it works as expected and have gone through the code.

The spacing issue made it a bit more difficult to find the changes (and make sure that nothing else changed), but I'm happy that it's all good.

**Comment by Wamacava** [comment]
Sorry for the spacing issue once again. It looks like IntelliJ forces us to use spaces instead of tabs, and it's a kind of good practice. We should use spaces if we want to earn more, as in the following article ;) https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/

I will add a commit that just updates all codebase to be consistent in tabs-spaces stuff so this doesn't hit us again in another pull request. I hope that's fine :)

**Comment by computer1978** [comment]
Hi Lukasz, It is new to me that spaces is preferable over tabs. Good work.

**Comment by computer1978** [comment]
I am willing to help if I know how to do it properly.

**Comment by acairns-345** [comment]
Sounds great, thank you!

**Comment by Wamacava** [comment]
It is quite a big discussion in the industry and I'm afraid there is not a good answer. See discussion here: https://softwareengineering.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e
I think that the main point is to be consistent with what you are using. But it's hard to be consistent with tabs when IntelliJ forces us to use spaces instead.
The work here is simply to go through all java files and click Ctrl+Alt+l for autoformatting. It's not a big job.

**Pull request merged**